src/main/java/com/alex/eshop/eshop/Entity/ItemEntity.java

```diff
@@ -0,0 +1,73 @@
+package com.alex.eshop.eshop.Entity;
+
+import javax.persistence.*;
+
+@Entity
+public class ItemEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private long id;
+    @Column(name = "name")
+    private String name;
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "category_id")
+    private CategoryEntity category;
+    @Column(name = "description")
+    private String description;
+    @Column(name = "image_src")
+    private String image_src;
```

Comment:
In Java we don't use underscores "_" in class/function/variable/etc names.

Wrong:
```
private String image_src;
```

Right:
```
private String imageSrc;
```
It's called camel-case convention.